packages/x-charts/src/internals/plugins/corePlugins/useChartDimensions/useChartDimensions.ts

```diff
@@ -197,18 +198,51 @@ export const useChartDimensions: ChartPlugin<UseChartDimensionsSignature> = ({
     [drawingArea.height, drawingArea.top],
   );
   const isPointInside = React.useCallback(
-    (x: number, y: number, targetElement?: Element) => {
+    (x: number, y: number, targetElement?: Element | EventTarget | null) => {
```

Comment:
Why did we add `EventTarget`? Isn't `Element` enough?

Reply:
> In all cases we use the target, we pass an EventTarget because the types can't know it is an element yet.

Right, that's why IMO the cast should be in the caller, because the caller knows if the cast is safe. 

Nevertheless, I don't think it's likely that we'll get an event target that isn't an element, so not a blocker to me.